pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -1146,6 +1183,31 @@ func podToGroupMember(pod *v1.Pod, includeIP bool) *controlplane.GroupMember {
 	return memberPod
 }
 
+func nodeToGroupMember(node *v1.Node) (member *controlplane.GroupMember) {
+	member = &controlplane.GroupMember{Node: node.Name}
+	nodeIPs, err := k8s.GetNodeAddrs(node)
```

Comment:
ditto, I think we should include transportAddresses as well if they are different from nodeIPs, otherwise it would be hard to explain why the policy doesn't apply to it. Maybe just use a sets.String to collect all IPs to deduplicate.
Perhaps it could extract a function to get all node IPs from a Node object and reuse it in nodeIPChanged function.

Reply:
Updated and add some unit test cases.